source/Calamari/Behaviours/PlanBehaviour.cs

```diff
@@ -43,7 +43,7 @@ protected override Task Execute(RunningDeployment deployment, Dictionary<string,
                 var resultCode = commandResult.ExitCode;
 
                 if (resultCode == 1)
-                    commandResult.VerifySuccess();
+                    cli.VerifySuccess(commandResult);
```

Comment:
This one is an interesting case, an exit code == 2 is actually ok, see https://www.terraform.io/docs/cli/commands/plan.html#other-options
![image](https://user-images.githubusercontent.com/122651/121477547-55ed5400-ca0b-11eb-957a-b99711c0c7c0.png)
So I think we need to handle this case a bit differently

Reply:
Yep, we only call VerifySuccess if the result code is 1 (line 45) so I believe it's handled already?

Reply:
Ah thinking about this more I think I see what you mean, if the result is 2 then we possibly won't log the untested version when we should. I'll add some extra handling.

Reply:
I've added some extra handling for exit code 2 to treat it as success and log untested versions if needed